float32 timestamp
float32 x
float32 y
float32 theta
float32 xVar
float32 yVar
float32 thetaVar